### PR TITLE
Linting updates

### PR DESCRIPTION
**Pull request by trypsynth**
### Link to issue number:
None
### Summary of the issue:
I know we've been trying to cut back on linting issues. I tried to tackle this a few months back, but didn't really know what I was doing. Let me know how it looks now.
### Description of how this pull request fixes the issue:
I linted a few files (buildVersion.py, XMLFormatting.py, wincon.py, and a few others). I've also created a new branch (lintingUpdates), if people wish to use it for future changes such as these.
### Testing strategy:
NVDA runs from source with no issues.
### Known issues with pull request:
None
### Change log entries:
B=None
### Code Review Checklist:
- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual testing.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers

### CODE REVIEW

File: tests/lint/flake8.ini [hunk 1]
```diff
@@ -4,7 +4,6 @@
 use-flake8-tabs = True
 # Not all checks are replaced by flake8-tabs, however, pycodestyle is still not compatible with tabs.
 use-pycodestyle-indent = False
-continuation-style = hanging
```
Comment (lukaszgo1) [inline tests/lint/flake8.ini:7]: Why you've removed this line?

File: source/vkCodes.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
-#vkCodes.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2007-2010 Michael Curran <mick@kulgan.net>, James Teh <jamie@jantrid.net>
+# vkCodes.py
```
Comment (lukaszgo1) [inline source/vkCodes.py:1]: This line, and all lines which just contains the file name should be removed according to the copyright header standards on the Wiki. Also it feels a bit weird  to me to modify the file just to reformat its copyright header so this should probably be reverted.

**Comment by TheQuinbox** [comment]
Ah, my bad. Fixing both now. :)

**Comment by TheQuinbox** [comment]
Hi, this should all be fixed now. :)

File: tests/lint/flake8.ini (hunk 1 above)
Comment (lukaszgo1) [reply tests/lint/flake8.ini:7]: This one is still unanswered.

**Comment by TheQuinbox** [comment]
Aaaah, my bad. Should be fixed now.

**Review by seanbudd — CHANGES_REQUESTED** [review]

**Pull request closed without merging**